docs/source/dataset_configuration/mixing_data_sources.md

```diff
@@ -0,0 +1,89 @@
+# Mixing Datasets
+
+Training a model often requires combining data from multiple different sources. Streaming makes combining these data sources, or streams, easy and configurable. See the [main concepts page](../getting_started/main_concepts.md#distributed-model-training) for a high-level view of distributed training with multiple streams.
+
+## Using Streams
+
+A stream is a data source, as a collection of shard files.Streams are represented by the {class}`streaming.Stream` object. Similar to {class}`streaming.StreamingDataset` itself, a `Stream` object can take in `remote` and `local` paths -- see [here](../getting_started/main_concepts.md#remote-data-streams) for an example.
```

Comment:
- missing space
- it's a collection of shards, each consisting of one or more shard files, which each have different forms like zip, raw, etc

Reply:
edited